FirebaseRemoteConfig/SwiftNew/ConfigExperiment.swift

```diff
@@ -18,8 +18,14 @@ import Foundation
 // TODO(ncooke3): Once everything is ported, the `@objc` and `public` access
 // can be removed.
 
+@objc(RCNConfigExperimentFake) public class ConfigExperimentFake: ConfigExperiment {
```

Comment:
```suggestion
// TODO(ncooke3): Move this definition to unit tests after they are ported.
@objc(RCNConfigExperimentFake) public class ConfigExperimentFake: ConfigExperiment {
```

Reply:
Alternatively, you could create the file put the fake in the unit tests now and still port the actual tests later.

Reply:
Good point, done.